### ISSUE TITLE
Missing argument for parameter 'propagationErrors' in call

### ISSUE THREAD

**Issue by JohannesNevels** [onevcat/Kingfisher#1352]
#### What

It will not compile and it's giving the error 'Missing argument for parameter 'propagationErrors' in call' as shown in the attachment.

![Schermafbeelding 2019-11-19 om 14 46 05](https://user-images.githubusercontent.com/10628176/69151658-56e22c00-0adb-11ea-9743-199f4d95e5d3.png)

![Schermafbeelding 2019-11-19 om 14 50 27](https://user-images.githubusercontent.com/10628176/69151975-f99aaa80-0adb-11ea-9dd9-8823f302e758.png)

![Schermafbeelding 2019-11-19 om 14 50 33](https://user-images.githubusercontent.com/10628176/69151988-fc959b00-0adb-11ea-9272-485fe1e0069f.png)

#### Reproduce

Xcode 10.3

Podfile
```
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '10.0'
pod 'Kingfisher', '~> 5.0'
```

**Comment by onevcat** [comment]
Seems that Xcode 10 (and Swift 5.0) does not generate memberwise initializer correctly for a pre-set variable. I will give it a fix soon.

**Comment by onevcat** [comment]
Should be fixed in 5.10.1.

**Comment by JohannesNevels** [comment]
I can confirm this, thnx! 🥇
